utils/getUserData.js

```diff
@@ -0,0 +1,22 @@
+async function getUserData(endPoint, username){
+  //   endPoint ---- string ----- always add a forward slash / in front of the endpoint url
+  //   username ------ string
+  // dynamic url
+  let url = endPoint + username;

```

Comment:
okay I see what you're trying to achieve here, but I feel using template strings would be a better way of constructing the url. This 👇 would've been better
```
let url = `/${endpoint}/${username}`
```

Reply:
On it bro